src/box/lua/config/source/file.lua

```diff
@@ -1,29 +1,30 @@
+local fio = require('fio')
+local log = require('log')
```

Comment:
Nit: We use `internal.config.utils.log` within the config code. It doesn't matter much here, but maybe it is good for consistency.